homeassistant/components/heos/media_player.py

```diff
@@ -235,10 +302,26 @@ async def async_update(self):
         controls = self._player.now_playing_media.supported_controls
         current_support = [CONTROL_TO_SUPPORT[control] for control in controls]
         self._supported_features = reduce(ior, current_support, BASE_SUPPORTED_FEATURES)
+        self._group = await self.async_get_group()
 
         if self._source_manager is None:
             self._source_manager = self.hass.data[HEOS_DOMAIN][DATA_SOURCE_MANAGER]
 
+    @log_command_error("unjoin_player")
+    async def async_unjoin_player(self):
+        """Remove this player from any group."""
+        if self.group_members:
+            controller = self.hass.data[HEOS_DOMAIN][DATA_CONTROLLER_MANAGER].controller
+            try:
+                await controller.create_group(self.player_id, [])
```

Comment:
```suggestion
                await controller.remove_group(self.player_id)
```

Reply:
`remove_group` is different, it requires a group id (which we do not have and we actually don't need it). I think you have pointed out in your first review that passing a single `player_id` to `set_group`/`create_group` properly removes any group for that player.